build.gradle

```diff
@@ -47,16 +47,18 @@ ext {
     minSdkVersion = 15
     targetSdkVersion = 28
     supportLibVersion = '1.1.0'
+    recyclerViewVersion = '1.2.0-alpha02'
```

Comment:
does this work? I was expecting that you'd need snapshot to get MergeAdapter, but paging will pull that in automatically